api/lib/domain/usecases/start-writing-campaign-results-to-stream.js

```diff
@@ -71,7 +48,7 @@ function _createHeaderOfCSV(skills, competences, areas, idPixLabel) {
       { title: `Acquis maitrisés du domaine ${area.title}`, property: `area_${area.id}_validatedSkillCount` },
     ])),
 
-    ...(_.map(skills, (skill) => ({ title: csvService.sanitize(skill.name), property: `skill_${skill.id}` }))),
+    ...(_.map(skills, (skill) => ({ title: `${skill.name}`, property: `skill_${skill.id}` }))),
```

Comment:
A priori `skill.name` est un string, donc on peut écrire plus simplement:
```suggestion
    ...(_.map(skills, (skill) => ({ title: skill.name, property: `skill_${skill.id}` }))),
```

Reply:
Bien vu. J'avais remis le code précédent l'ajout de `csvService.sanitize` qui contenait ce template mais il est effectivement superflu.